content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -78,55 +78,56 @@ The latency, throughput, and the proxies' CPU and memory consumption are measure
 ### Sidecar CPU and memory usage
 
 Since the sidecar proxy performs additional work on the data path, it consumes CPU
-and memory. In Istio 1.22, a sidecar proxy consumes about 0.5 vCPU per 1000
-requests per second.
+and memory. In Istio 1.24, with 1000 http requests per second containing 1 KB of payload each
+- a sidecar proxy with 4 worker threads consumes about 27% CPU and 61 MB of memory.
```

Comment:
Can we avoid using % and instead use 0.25 vCPU (note I also rounded -- its not like our measurements are so precise we can really commit to that level of granularity).

We can also likely round memory to 50mb.

Also, why 4 worker threads? The default (and 99.999% of users) run 2 threads